input/about/election/campaign-2019/_directory.yaml

```diff
@@ -0,0 +1,2 @@
+﻿Analyzers:
+  - ValidateRelativeLinks=None
```

Comment:
This whole `campaign-2019` folder had links that wouldn't validate and it appears to be here for historical purposes so I disabled the validation for these files